.github/workflows/ci.yml

```diff
@@ -20,10 +28,10 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '8.0'
+          php-version: ${{matrix.php}}
 
       - name: Install dependencies
-        run: composer require "laravel/framework:^${{matrix.laravel}}.0"
+        run: composer require "laravel/framework:${{matrix.laravel}}"
```

Comment:
I'd keep the original syntax, the workflow name is more readable then (`L10` instead of `L^10`). With Laravel we don't care about specifying the minor version, so there's no need to specify versions as `^x` in the matrix.